supervision/detection/core.py

```diff
@@ -488,6 +493,46 @@ def from_roboflow(cls, roboflow_result: dict) -> Detections:
             tracker_id=trackers,
         )
 
+    @classmethod
+    def from_inference(
```

Comment:
If we want to rename `from_roboflow` to `from_inference` let's mark `from_roboflow` as [deprecated](https://github.com/roboflow/supervision/blob/5b7e0bac5afa1c4aaf47d75fa58d60bf92d032a2/supervision/utils/internal.py#L5).